helm/cmsweb/crabcache/templates/configMap.yaml

```diff
@@ -34,7 +34,9 @@ metadata:
 data:
   setup-certs-and-run.sh: |
     #!/bin/bash
-    cp /host/etc/grid-security/* /etc/grid-security
+    sudo cp /host/etc/grid-security/* /etc/grid-security
     echo 'INFO Files in /etc/grid-security'
     ls -lahZ /etc/grid-security
     su  -c "cd /data && /data/run.sh" --preserve-environment  _crabcache
```

Comment:
@arooshap 
This line  `su  -c "cd /data && /data/run.sh" --preserve-environment  _crabcache` should be commented.

Reply:
https://github.com/arooshap/CMSKubernetes/blob/a70bf64fb9597f12e8e5cddb0b166c0a7d2f2f9d/kubernetes/cmsweb/services/crabcache.yaml#L57

This needs to be updated as well.

Reply:
yes, I have fixed there as well. Thanks